index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
added

Reply:
Great work living comment as required but there are some changes you still need to perform.
Kindly, I suggest you use polite suggestions, like: `Great work using semantic tags`.